Model/ReachConfigProvider.php

```diff
@@ -130,9 +130,16 @@ protected function getConsumerIp()
      */
     protected function checkLocalIP($ip)
```

Comment:
can we put this checkLocalIP in the helper file rather than here and in Currency.php? if it's outside of the scope of this ticket that's ok

Reply:
It should be in a helper file as it is in minimum two different files in our code base. 
But i think that type of code duplication cleanup can be handled as a separate ticket.

Reply:
https://withreach.atlassian.net/browse/MAG-110 created a ticket, thanks!